pkg/proxy/servicechangetracker_test.go

```diff
@@ -86,6 +87,13 @@ func makeServicePortName(ns, name, port string, protocol v1.Protocol) ServicePor
 		Protocol:       protocol,
 	}
 }
+func makeIPs(ipStr ...string) []net.IP {
+	var ips []net.IP
+	for _, s := range ipStr {
+		ips = append(ips, net.ParseIP(s))
```

Comment:
verify is going to complain about this; you have to always use `netutils.ParseIPSloppy`.

also, given that every caller of this function only passes a single IP, it might be simpler to just do them all inline? `[]string{"10.1.2.4"}` → `[]net.IP{netutils.ParseIPSloppy("10.1.2.4")}` ?

Reply:
Updated, but I kept the function. The use of a slice indicates that there *may* be more than one IP, and then the function will be handy